src/pages/mission/entry/components/MissionInfoCard.tsx

```
@@ -0,0 +1,160 @@
+        <svg
+          width="17"
+          height="24"
+          viewBox="0 0 17 24"
+          fill="none"
+          className="text-moamoa-200"
+        >
+          <path
+            d="M2 2L15 12L2 22V2Z"
+            fill="currentColor"
+            stroke="currentColor"
+            strokeWidth="2"
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)